tests/test_reservation.py

```diff
@@ -44,4 +45,9 @@ def test_reservation_duration():
         comment=comment,
         status=status
     )
-    assert new_reservation.duration() == end_time - start_time
\ No newline at end of file
+    assert new_reservation.duration() == end_time - start_time
+
+def test_retrieve(client):
+    response = client.post("/reservations/retrieve", data={"location" : 1, "date" : datetime(1, 1, 1)})
+    assert (response.status_code == HTTPStatus.OK
```

Comment:
@carlosmiranda one assert for one test pls, we've discussed it in telegram channel.
`assert response.status_code == HTTPStatus.OK`
`assert response.is_json`